src/test/java/com/wikia/webdriver/testcases/signuptests/SignUpTests.java

```diff
@@ -222,4 +224,21 @@ public void SignUp_008_signupJapaneseUser() {
     preferences.selectTab(tabNames.EMAIL);
     preferences.verifyEmailMeSection();
   }
+
+  @BeforeMethod(alwaysRun = true)
+  public void setEnvironmentCredentials() {
```

Comment:
Two the same instances of a code block indicates that the code could be extracted into one place. When it comes to environments, this case indicates that our repository lacks an environment manager that could be used to handle this case. I'd recommend to start it here and do the following:
1. Create "environmentManager" class
2. move setEnvironmentCredentials method to that class
3. invoke the method in each of the two Test class 

I'd then recommend to knowledge share the creation of such manager so that anyone can reuse it when he gets to environment related scenario. Such cases occured in the past and I expect them to occur in the future as well